README.md

```diff
@@ -41,6 +41,16 @@ filebeat-v6.1.3-go1.10rc1-linux-amd64
 firehose.so-1-snapshot-v6.1.3-go1.10rc1-linux-amd64
 ```
 
+### Trouble-shooting
+
+If you see `No such file or directory` error of filebeat while building the plugin, you likely to be relying on the default `GOPATH`. It is `$HOME/go` in recent versions of golang.
+
+If you got affected by this, try running:
+
+```
+$ make BEATS_VERSION=v6.1.3 GOPATH=$HOME/go
```

Comment:
To be extra clear, without setting GOPATH explicitly, [this](https://github.com/s12v/awsbeats/blob/7ed4a0eefb493eeed3260520b8d5330d81a4109c/Makefile#L270):

```
@cd "$$GOPATH/src/github.com/elastic/beats/filebeat"
```


results in:

```
@cd "/src/github.com/elastic/beats/filebeat"
```

which ends up with the `No such file or directory` error to me and probably many people.

That's why I'm suggesting to set `GOPATH=$HOME/go` here.